platform-hub-api/app/serializers/kubernetes_token_serializer.rb

```diff
@@ -1,5 +1,31 @@
-class KubernetesTokenSerializer < KubernetesTokenBaseSerializer
-  attribute :expire_privileged_at, if: -> { object.expire_privileged_at.present? }  do
+class KubernetesTokenSerializer < BaseSerializer
+  belongs_to :cluster
+
+  attributes(
+    :id,
+    :kind,
+    :name,
+    :uid,
+    :groups
+  )
+
+  attribute :token do
+    object.decrypted_token
+  end
+
+  attribute :user, if: -> { object.user.present? } do
+    {
+      id: object.user.id,
+      name: object.user.name,
+      email: object.user.email
+    }
+  end
+
+  attribute :description, if: -> { object.robot? } do
+    object.description
+  end
+
+  attribute :expire_privileged_at, if: -> { object.expire_privileged_at.present? } do
```

Comment:
Likewise here you probably don't need the block (I know this was in the previous code, but that may have been an oversight).